packages/expo-av/src/AV.types.ts

```diff
@@ -8,56 +8,57 @@ export enum PitchCorrectionQuality {
   High = ExponentAV && ExponentAV.Qualities && ExponentAV.Qualities.High,
 }
 
-export type AVPlaybackSource =
-  | number
-  | {
-      uri: string;
-      overrideFileExtensionAndroid?: string;
-      headers?: { [fieldName: string]: string };
-    }
-  | Asset;
+export type AVPlaybackSource = number | AVPlaybackNativeSourceAndroid | Asset;
 
 export type AVPlaybackNativeSource = {
   uri: string;
   overridingExtension?: string | null;
   headers?: { [fieldName: string]: string };
 };
 
+export type AVPlaybackNativeSourceAndroid = {
```

Comment:
After investigating the code a bit deeply, I'm a bit confused, why this object differs from `AVPlaybackNativeSource` in the first place, and I want to investigate this a bit further. Also it looks like the current name might be a bit misleading, since it came from the field name differences, not from the actual functionality. 

So, let me try to dive into this again, before we merge it.